test/DynamoCoreTests/DSEvaluationModelTest.cs

```diff
@@ -1187,6 +1187,20 @@ public void TestHeterogenousList()
             AssertPreviewValue(guidY, new object[] { null, null, null, 10.2 });
             AssertPreviewValue(guidZ, new object[] { null, null, null, 15.2 });
         }
+
+        [Test, Category("UnitTests")]
+        public void ReplicationWithEmptySubLists()
```

Comment:
@reddyashish could you also add a language test (using DS code) in ProtoTest.dll?

Reply:
Added this.